### PR TITLE
34 release notes

### PR DESCRIPTION
**Pull request by BillDett**
First draft- needs known issues added following QE testing.

### CODE REVIEW

File: modules/rn_3_40.adoc [hunk 1]
```diff
@@ -0,0 +1,128 @@
+[[rn-3-400]]
+== Version 3.4.0
+
+=== quay / clair-jwt / quay-builder / clair
+
+Added/Changed:
+
+* Clair V4 now GA and the default security scanner for Quay 3.4.0.  New features include support for notifications and disconnected deployments.
+* New ConfigTool replaces the older Config App, providing better configuration validation and integration with the new Quay Operator.  Quay now uses same validator as the Config Tool at start time to ensure its configuration is correct.
+* Quay Builders are now supported only on OpenShift/Kubernetes and utilize Fedora CoreOS for builder sandbox VMs.  Quay Build Manager completely re-written to use gRPC and address numerous issues.
```
Comment (thomasmckay) [inline modules/rn_3_40.adoc:10]: s/Fedora CoreOS/Red Hat CoreOS (RHCOS)/

File: modules/rn_3_40.adoc [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+* Quay codebase now completely migrated to python 3 with numerous dependency updates.
+* (Tech Preview) Support for Helm V3 is no longer considered experimental.  It can be enabled as follows:
++
+```
+# Enable Helm support- requires that general OCI support (Tech Preview) is enabled.
+FEATURE_GENERAL_OCI_SUPPORT = True
+FEATURE_HELM_OCI_SUPPORT = True
```
Comment (thomasmckay) [inline modules/rn_3_40.adoc:17]:
```
FEATURE_GENERAL_OCI_SUPPORT: true
FEATURE_HELM_OCI_SUPPORT: true
```

File: modules/rn_3_40.adoc [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+* PROJQUAY-1011	Accessing build logs from super user panel doesnt work
+* PROJQUAY-1015	RPM command error when getting rpm packages from layer database    
+* PROJQUAY-1023	oraclelinux:7 causes matcher bug    
+* PROJQUAY-1035	Unable to override gunicorn worker count in k8s
+* PROJQUAY-1087	Fail to pull from managed objectstorage
+* PROJQUAY-1101	Typo in /tools/generatekeypair.py
+* PROJQUAY-1103	Remove need to modify SCC 
+* PROJQUAY-1112	Quay database reaches connection limit
+* PROJQUAY-1122	Specify pull secret for component images
+* PROJQUAY-1132 running as config should not try to set httppasswd
```
Comment (thomasmckay) [inline modules/rn_3_40.adoc:76]: Whitespace cleanup (or replace tabs with whitespace?)

File: modules/rn_3_40.adoc [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+Known Issues:
+
+* <TODO>
+
+
+=== quay-container-security-operator
+
+
+* Only supported on OCP-4.2 or newer
```
Comment (thomasmckay) [inline modules/rn_3_40.adoc:113]: OCP N-2 is 4.5

File: modules/rn_3_40.adoc [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+
+Fixed:
+
+* PROJQUAY-676  Wrong image vulnerabilities link in OCP4.4 Overview page
+
+
+=== quay-openshift-bridge-operator
+
+* Only supported on OCP-4.2 or newer
```
Comment (thomasmckay) [inline modules/rn_3_40.adoc:122]: OCP N-2 is 4.5

**Review by thomasmckay — CHANGES_REQUESTED** [review]

File: modules/rn_3_40.adoc [hunk 6]
```diff
@@ -0,0 +1,128 @@
+[[rn-3-400]]
+== Version 3.4.0
+
+=== quay / clair-jwt / quay-builder / clair
+
+Added/Changed:
+
+* Clair V4 now GA and the default security scanner for Quay 3.4.0.  New features include support for notifications and disconnected deployments.
+* New ConfigTool replaces the older Config App, providing better configuration validation and integration with the new Quay Operator.  Quay now uses same validator as the Config Tool at start time to ensure its configuration is correct.
```
[the remark this answers is not in the document]
Comment (BillDett) [reply modules/rn_3_40.adoc:9]: no- just pointing out that Quay now uses the same config validation at startup time as our ConfigTool (Jonathan's nifty table).  Perhaps I haven't made that clear enough- I can add some more words.

File: modules/rn_3_40.adoc (hunk 1 above)
Comment (BillDett) [reply modules/rn_3_40.adoc:10]: good catch

File: modules/rn_3_40.adoc (hunk 2 above)
Comment (BillDett) [reply modules/rn_3_40.adoc:17]: ACK

File: modules/rn_3_40.adoc (hunk 3 above)
Comment (BillDett) [reply modules/rn_3_40.adoc:76]: ACK

File: modules/rn_3_40.adoc (hunk 4 above)
Comment (BillDett) [reply modules/rn_3_40.adoc:113]: ACK

File: modules/rn_3_40.adoc (hunk 5 above)
Comment (BillDett) [reply modules/rn_3_40.adoc:122]: ACK

**Comment by BillDett** [comment]
Closing this PR as it's gone pretty stale- raised final changes via https://github.com/quay/quay-docs/pull/113

**Pull request closed without merging**